app/views/groups/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1 class="text-center"> Categories </h1>
+<div class="container">
+  <%= render 'categories' %>
+</div>
+
+<footer class="footer" >
+  <div class="fixed-bottom d-flex justify-content-center" style="height:60px">
+    <%= link_to "Add a new category", new_group_path, class: "btn btn-primary btn-large btn-success ", style:"width:60%"%>
+  </div>
+</footer>
```

Comment:
- [ ] Sometimes the category image is too big and takes up most of the screen, it would be better to make it smaller in order to be able to see the items without scrolling down:

<img width="379" alt="Screen Shot 2022-08-13 at 2 27 13 PM" src="https://user-images.githubusercontent.com/83717285/184508052-0499cdb1-af12-4503-b85a-624f4b0f0b76.png">

Reply:
Thanks, I will take care that.